alacritty_terminal/src/input.rs

```diff
@@ -707,15 +707,15 @@ impl<'a, A: ActionContext + 'a> Processor<'a, A> {
         }
 
         let point = self.ctx.terminal().visible_to_buffer(point);
-        let url = self.ctx.terminal().urls().drain(..).find(|url| url.contains(point))?;
+        let url = self.ctx.terminal().urls().iter().find(|url| url.contains(point))?;
         let text = self.ctx.terminal().url_to_string(&url);
 
         let launcher = self.mouse_config.url.launcher.as_ref()?;
         let mut args = launcher.args().to_vec();
         args.push(text);
 
         match start_daemon(launcher.program(), &args) {
-            Ok(_) => debug!("Launched {} with args {:?}", launcher.program(), args),
+            Ok(_) => println!("Launched {} with args {:?}", launcher.program(), args),
```

Comment:
I'd assume this `println` is for debugging? It should be removed before merging.

Reply:
Oh, I missed that, thanks